source/system/String.ooc

```diff
@@ -311,8 +311,9 @@ strArrayFromCString: func ~hack (argc: Int, argv: String*) -> String[] {
 }
 cStringPtrToStringPtr: func (cstr: CString*, len: Int) -> String* {
 	toRet: String* = calloc(len, Pointer size) // otherwise the pointers are stack-allocated
-	for (i in 0 .. len) {
+	for (i in 0 .. len)
 		toRet[i] = makeStringLiteral(cstr[i], cstr[i] length())
-	}
 	toRet
 }
+
+//GlobalCleanup register(|| String free~all())
```

Comment:
It crashes because something in Fixture is trying to use string literals after the cleanup function is called.
Comment out the content of `printFailures` and it will not crash on this branch.
For me it crashed in this line 
```
This _print(t"Total time: %.2f s\n" format(This totalTime))
```